pkg/restapi/vc/operation/operations_test.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package operation
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testCreateCredentialRequest = `{
+"context":"https://www.w3.org/2018/credentials/examples/v1",
+"type": [
+    "VerifiableCredential",
+    "UniversityDegreeCredential"
+  ],
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "BachelorDegree",
+      "university": "MIT"
+    },
+    "name": "Jayden Doe",
+    "spouse": "did:example:c276e12ec21ebfeb1f712ebc6f1"
+  },
+
+  "issuer": {
+    "id": "did:example:76e12ec712ebc6f1c221ebfeb1f",
+    "name": "Example University"
+  }
+}`
+)
+
+func TestCreateCredentialHandler_InvalidJSON(t *testing.T) {
+	op := New()
+
+	createCredentialHandler := getHandler(t, op, createCredentialEndpoint)
+
+	req, err := http.NewRequest(http.MethodPost, "/credentials", bytes.NewBuffer([]byte("")))
+	require.NoError(t, err)
+
+	rr := httptest.NewRecorder()
+
+	createCredentialHandler.Handle().ServeHTTP(rr, req)
+
+	require.Equal(t, http.StatusBadRequest, rr.Code)
```

Comment:
Something else to check out...it looks like your handler will always return StatusCreated, so I'm curious why this test actually passes. (The subsequent call to WriteHeader is ignored?)

Reply:
updated